libpod/container.go

```diff
@@ -965,8 +969,36 @@ func (c *Container) Inspect(size bool) (*ContainerInspectData, error) {
 
 // Commit commits the changes between a container and its image, creating a new
 // image
-func (c *Container) Commit() (*storage.Image, error) {
-	return nil, ErrNotImplemented
+func (c *Container) Commit(pause bool, options CopyOptions) error {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+
+	if err := c.syncContainer(); err != nil {
+		return err
+	}
+
+	if c.state.State == ContainerStateRunning && pause {
+		if err := c.runtime.ociRuntime.pauseContainer(c); err != nil {
+			return errors.Wrapf(err, "error pausing container %q", c.ID())
+		}
+		defer func() {
+			if err := c.runtime.ociRuntime.unpauseContainer(c); err != nil {
+				logrus.Errorf("error unpausing container %q: %v", c.ID(), err)
+			}
+		}()
+	}
+
+	tempFile, err := ioutil.TempFile("/var/tmp", "kpod-commit")
```

Comment:
Can you make this c.runtime.config.TmpDir instead of /var/tmp?

Reply:
fixed